$OpenBSD: patch-include_xcursor_h,v 1.1 2011/12/01 16:00:01 dcoppa Exp $

Fix prototype (git commit 0ea64ae404c38391aa50ec91dbcebdbda9cac833)

--- include/xcursor.h.orig	Thu Dec  1 13:31:31 2011
+++ include/xcursor.h	Thu Dec  1 13:32:46 2011
@@ -35,6 +35,6 @@ int xcursor_get_xcb_cursor(enum xcursor_cursor_t c);
  * races might occur (even though we flush the Xlib connection).
  *
  */
-void xcursor_set_root_cursor();
+void xcursor_set_root_cursor(int cursor_id);
 
 #endif
